SizeConverter.cs

```diff
@@ -1,49 +1,49 @@
 ﻿namespace p2pcopy
 {
-    internal static class SizeConverter
+    internal enum DataSizeUnit

```

Comment:
I really prefer it INSIDE the class to restrict its scope. That's something I use a lot, but ok, all the other issues where there.

Reply:
good point; when I made this change I didn't realise that this enum was only used in this fie; I can put it back nested if you want